### DISCUSSION TITLE
The best strategies for spelling, word by word

### DISCUSSION THREAD

**Discussion by YusufCandraArif** [SYSTRAN/faster-whisper#904] [Q&A]
What is the best strategy for audio to text on word by word pronunciation. The words spoken are very short, but some words that sound similar can be mistaken.

**Comment by toanhuynhnguyen** [comment]
I actually do not understand your question, can you please describe more? With the Faster-Whisper, you can transcribe an audio file into a transcript at words level.

**Reply by YusufCandraArif** [reply]
ok sir. So I want to extract syllables from audio. The difficulty is because the audio is very short.
what is the best setting in faster-whisper for this kind of thing?
